selection_tools/prepare_histograms.C

```diff
@@ -168,7 +204,6 @@ int main(int argc, char *argv[])
         fill_first_mother_hists(mother_2);
```

Comment:
So here I guess we should have 2x more histograms, separately for opposite sign and same sign, right?